<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Upload</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
    background-color: rgb(211,211,211);
    color: rgb(88, 88, 90);
    selection-background-color: rgb(21, 165, 137);
    selection-color: rgb(245, 244, 244);
    font: 16pt
}

QProgressBar::chunk
{
  background-color: rgb(21, 165, 137);
}

QLabel#error_label
{
    color: rgb(220, 130, 30);
}

QLineEdit, QTextEdit, QTableWidget
{
   background-color:  rgb(245, 244, 244)
}

QPushButton
{
	background-color: rgb(21, 165, 137);
    color: rgb(245, 244, 244);
}

QPushButton#home_button, QPushButton#parent_button, QPushButton#refresh_button
{
    background-color: rgb(245, 244, 244);
}

QTabWidget#info_tabs
{
     background-color: background-color: rgb(211,211,211);
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QLabel" name="destination_label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="overwrite">
     <property name="text">
      <string>Overwrite existing data</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Uploading to</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Upload data</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Options:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="folder_button">
       <property name="text">
        <string>Select Folders</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="file_button">
       <property name="text">
        <string>Select Files</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="9" column="0" colspan="6">
    <widget class="QLabel" name="error_label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="upload_button">
       <property name="text">
        <string>Upload</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="hide_button">
       <property name="text">
        <string>Close Window</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <widget class="QTextBrowser" name="sources_list"/>
   </item>
   <item row="5" column="2">
    <widget class="QProgressBar" name="progress_bar">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
